core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -61,6 +61,13 @@
     <div class="oppia-login-required" ng-if="!$ctrl.userDetailsLoading && !$ctrl.userIsLoggedIn">
       <login-required-message></login-required-message>
     </div>
+    <div class="oppia-dashboard-topic-container"
```

Comment:
To be consistent with the topic selector in the "Translate Text" tab, can we move this to the header and format it to look like the current filter (change "Filter by Topics" font/size, and move selector to right of "Review rights")?

Reply:
Well maybe you can just add the translation review tab to `showTopicSelector()`, as it seems it's the same logic as https://github.com/oppia/oppia/pull/13539/files#diff-5212414c461bb4033898d424a6baa626fcd7ded8b08e719a4619ae9647d65e22.

Reply:
done
![Screen Shot 2022-03-07 at 2 04 30 PM](https://user-images.githubusercontent.com/35385366/157135223-42eee5d2-666b-477b-b082-16e7ac884d1d.png)

Reply:
![Screen Shot 2022-03-07 at 1 52 54 PM](https://user-images.githubusercontent.com/35385366/157135288-29d4acd0-8e3e-4c82-acd6-2bc62cd44d22.png)

Reply:
done